docs/integration-tutorial.md

```diff
@@ -0,0 +1,212 @@
+- [Synthetix Integration](#synthetix-liquidity-bot)
+  - [Getting Started](#Getting-Started)
+  - [Writing a Bot Script](#Write-the-Script)
+  - [Build a Docker Image](#build-a-docker-image)
+  - [Running the Bot](#running-the-bot)
+- [[Optional] Testing Locally](#Testing-Locally)
+
+## Requirements
+
+- [Truffle](https://www.trufflesuite.com/docs/truffle/getting-started/installation)
+- [Node](https://nodejs.org/en/)
+- [Yarn](https://yarnpkg.com/) or alternatily you could use the node package manager `npm` included with Node.
+
+To follow this tutorial more interactively, please clone the following github repository containing all the relevant code being referenced.
+
+```sh
+git clone https://github.com/gnosis/dex-integration-tutorial
+cd dex-integration-tutorial
+```
+
+This guide will focus primarily on interacting with dex-contracts (i.e. the Gnosis Protocol Smart contracts) and writing a minimally functional integration with synthetix exchange. Note that the README in the cloned repository may be updated with additional integrations (such as Uniswap) in the future.
+
+## Project Initialization and Network Configuration
+
+This section is intended to be a first step when building a new project from scratch. If you would prefer to start from a preconfigured environment, feel free to skip ahead to the next section.
+
+```sh
+mkdir <project_title>
+cd project_title
+truffle init
+yarn add @truffle/contract
+yarn add @gnosis.pm/dex-contracts
+[optional] yarn add @gnosis.pm/util-contracts
+[optional] yarn add @openzeppelin/contracts@2.5.1
+```
+
+Make sure that you have a valid (truffle configuration)[https://www.trufflesuite.com/docs/truffle/reference/configuration] for the network you intend on interacting with.
+We will be using the common `truffle-config.js` found in most, if not all, gnosis smart contract projects.
+In order to use the common gnosis truffle configuration, you will need to install the optional package `@gnosis.pm/util-contracts` listed above.
+Furthermore, you will likely have to provide your own `INFURA_KEY`
+
+Note that, if you plan to be experimenting with a locally hosted development network, you will need to install additional "devDepencencies" to mirgrate the `BatchExchange` Smart Contracts. This will be covered in detail once we have successfully confirmed our ability to interact with the existing mainnet smart contracts.
+
+## Interaction with Batch Exchange
+
+Assuming you were successful with optionally configuring your own project from the section above, we will continue here from a pre-configured environment obtained and installed as follows:
+
+```sh
+git clone git@github.com:bh2smith/dex-integration-tutorial.git
+cd dex-integration-tutorial
+yarn
+```
+
+This should put us in the same place as has having completed the project initialization steps independantly.
+We are now prepared to start scripting interactions with the Gnosis Protocol.
+To test this run the `exchange_interaction` script via
```

Comment:
Here is a break in the reading logic

Reply:
I am not sure I understand this comment.

Reply:
Since the sentence ends with `via:` one expects the actual command and not a new sentence `Note`. But maybe, its just me...